RoddyCore/test/src/de/dkfz/roddy/knowledge/files/BaseFileTest.groovy

```diff
@@ -72,33 +75,38 @@ class BaseFileTest {
         FileSystemAccessProvider.initializeProvider(true)
         //Setup plugins and default configuration folders
         LibrariesFactory.initializeFactory(true)
-        LibrariesFactory.getInstance().loadLibraries(LibrariesFactory.buildupPluginQueue(LibrariesFactoryTest.callLoadMapOfAvailablePlugins(), "DefaultPlugin").values() as List)
-        ConfigurationFactory.initialize(LibrariesFactory.getInstance().getLoadedPlugins().collect { it -> it.getConfigurationDirectory() })
+        LibrariesFactory.instance.loadLibraries(LibrariesFactory.
+                buildupPluginQueue(LibrariesFactoryTest.callLoadMapOfAvailablePlugins(), 'DefaultPlugin').values() as List)
+        CF.initialize(LibrariesFactory.instance.getLoadedPlugins().collect { it -> it.getConfigurationDirectory() })
 
-        final Configuration mockupConfig = new Configuration(new PreloadedConfiguration(null, Configuration.ConfigurationType.OTHER, DEFAULT, "", "", null, "", ResourceSetSize.l, null, null, null, null), ConfigurationFactory.getInstance().getConfiguration(DEFAULT)) {
+        final Configuration mockupConfig = new Configuration(new PreloadedConfiguration(null, Configuration.ConfigurationType.OTHER, DEFAULT,
+                '', '', null, '', ResourceSetSize.l, null, null, null,
+                null), CF.instance.getConfiguration(DEFAULT)) {
             @Override
             File getSourceToolPath(String tool) {
-                if (tool == "wrapinScript")
+                if (tool == 'wrapinScript')
                     return super.getSourceToolPath(tool)
-                return new File("/tmp/RoddyTests/RoddyTestScript_ExecutionServiceTest.sh")
+                return new File('/tmp/RoddyTests/RoddyTestScript_ExecutionServiceTest.sh')
             }
         }
-        syntheticTestFileClass = LibrariesFactory.getInstance().loadRealOrSyntheticClass("FileWithFileStage", BaseFile.class as Class<FileObject>)
-        ToolEntry toolEntry = new ToolEntry("RoddyTests", "RoddyTests", "RoddyTestScript_ExecutionServiceTest.sh")
-        toolEntry.getOutputParameters(mockupConfig).add(new ToolFileParameter(syntheticTestFileClass, null, "TEST", new ToolFileParameterCheckCondition(true)))
+        syntheticTestFileClass = LibrariesFactory.getInstance().loadRealOrSyntheticClass('FileWithFileStage', BaseFile.class as Class<FileObject>)
```

Comment:
Assign LibrariesFactory.instance to a field

Reply:
Improvements are only marginal. Note that the factory initializers of CF and LF are different: CF does not return the singleton and LF's has a lengthly name. We could introduce ConfigurationFactory.initializeFactory (lengthly) to return a value.

What actually about importing the CFs static fields and static methods and safe even the chars for the CF. prefix?